### DISCUSSION TITLE
Definition of roads - LCZ classification (OSM)

### DISCUSSION THREAD

**Discussion by EmmanuelleKerjouan** [orbisgis/geoclimate#903] [General]
I work in the Montpellier metropolitan area and I've noticed for OSM, that the roads around small towns are not modeled. When you go to OSM, some of these roads exist and are tagged "minor road". 
Here's the OSM link for this area: https://www.openstreetmap.org/query?lat=43.5162&lon=3.8578#map=17/43.51595/3.85763
![roads_OSM](https://github.com/orbisgis/geoclimate/assets/126885597/1f11481a-720f-455c-a1bd-409786c8efb7)

**Comment by ebocher** [comment]
Thanks I wlll check

**Reply by ebocher** [reply]
this result is due to the fact that we do not take into account the access type = "agricultural" and  "forestry"

**Reply by EmmanuelleKerjouan** [reply]
Okay, I understand better, thank you very much.
